lib/core/src/Cardano/Wallet/Primitive/Types/TokenPolicy.hs

```diff
@@ -81,3 +83,11 @@ instance ToText TokenName where
 
 instance FromText TokenName where
     fromText = pure . UnsafeTokenName
```

Comment:
Related question: are we planning to change the type of `TokenName` from `Text` to `ByteString` (or perhaps `ShortByteString`), given that it can be any sequence of bytes between 0 bytes and 32 bytes in length (and therefore might not be representable by `Text`).

Reply:
Of course, because Text is the wrong type.

Reply:
Fixed.